core/web/cors_test.go

```diff
@@ -32,6 +26,10 @@ func TestCors_DefaultOrigins(t *testing.T) {
 
 	for _, test := range tests {
 		t.Run(test.origin, func(t *testing.T) {
+			app := cltest.NewApplicationWithConfig(t, config, cltest.Unstarted)
+
+			client := app.NewHTTPClient()
+
```

Comment:
Subtests each need their own, because a `client.Get()` failure triggers on the inherited `*testing.T` from the `app`, and they are only allowed to fail their own `T` - not the parent's.